site/content/docs/5.1/components/alerts.md

```diff
@@ -130,6 +130,31 @@ You can see this in action with a live demo:
 When an alert is dismissed, the element is completely removed from the page structure. If a keyboard user dismisses the alert using the close button, their focus will suddenly be lost and, depending on the browser, reset to the start of the page/document. For this reason, we recommend including additional JavaScript that listens for the `closed.bs.alert` event and programmatically sets `focus()` to the most appropriate location in the page. If you're planning to move focus to a non-interactive element that normally does not receive focus, make sure to add `tabindex="-1"` to the element.
 {{< /callout >}}
 
+### Dark background
+
+To get the inverted alert behavior, simply add `.bg-dark` class to your alert.
```

Comment:
Proposal:
```markdown
Add `.bg-dark` to the `.alert` for a dark variant. Close button can be inverted as well by using the [dark variant of close button]({{< docsref "/components/close-button#dark-variant" >}})
```

And so it would mean renaming https://boosted.orange.com/docs/5.1/components/close-button/#white-variant in "Dark variant" instead of "White variant".